spec-0006/index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "SPEC 6 — Keys to the Castle"
+date: 2022-12-19
+author:
+  - "Stéfan van der Walt <stefanv@berkeley.edu>"
+  - "Jarrod Millman <millman@berkeley.edu>"
+discussion:
+endorsed-by:
+---
+
+## Description
+
+<!--
+Briefly and clearly describe the proposal.
+Explain the general need and the advantages of this specific proposal.
+If relevant, include examples of how the new functionality would be used,
+intended use-cases, and pseudo-code illustrating its use.
+-->
+
```

Comment:
@stefanv can you add that the starting principle is: don't use passwords unless you absolutely cannot avoid it? Servers should use SSH keys, PyPI, GitHub etc. all have concepts of Teams, etc.